### ISSUE TITLE
Issue after update

### ISSUE THREAD

**Issue by perariontaeadastra** [joue-quroi/spoof-timezone#6]
Hi,
after the latest update, version 0.2.0, the extension does not obey static settings, i.e. it does not allow to show real system time, it shows only CEST, which interferes with some websites, where spoofing should not apply. Setting it manually does not help, as does not help switching off by hitting the icon. The only thing that helps is disabling the extension in the browser settings.
Anyways, thank you for a great job!
With regards, Alex.

**Comment by bjmiller121** [comment]
I'm seeing this as well. I would expect clicking the icon to disable the spoofing, but it seems to keep spoofing regardless of the "state" of the icon. I have to disable the extension to stop the spoofing.

**Comment by joue-quroi** [comment]
True! so to have our spoofer module installed on all web pages before the actual web page loads, I switched to content script injection method which guarantees that our script gets executed before the document is loaded. The drawback is that there is no way to read local preferences synchronously and hence we are going to spoof the `Date` object anyway. So since there is no reliable method to perform the injection other than the hard-coded content script method which works on all browsers, for now, it is not possible to disable the extension. I hope Chrome supports `browser.contentScript` soon and this would fix this issue and would make the whitelisting possible as well. I just added a notification to the browser action button to inform users.
